packages/shared/src/components/cards/SimilarPosts/PostEngagementCounts.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { singleLineClamp } from '../../utilities';
+
+interface PostEngagementCountsProps {
+  upvotes: number;
+  comments: number;
+  className?: string;
+}
+
+export function PostEngagementCounts({
+  upvotes,
+  comments,
+  className,
+}: PostEngagementCountsProps): ReactElement {
+  return (
+    <p
+      className={classNames('typo-footnote', singleLineClamp, className)}
+      data-testid="post-engagements-count"
+    >
+      {upvotes ? `${upvotes} Upvotes` : ''}
+      {upvotes && comments ? <> &#x2022; </> : ''}
```

Comment:
Did you mean to use `separatorCharacter` here?

Reply:
Oh, yep. Forgot to utilize the variable here.